browser/ui/webui/brave_rewards_ui.h

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_UI_WEBUI_BRAVE_REWARDS_UI_H_
+#define BRAVE_BROWSER_UI_WEBUI_BRAVE_REWARDS_UI_H_
+
+#include "brave/browser/ui/webui/basic_ui.h"
+
+class BraveRewardsUI : public BasicUI {
+ public:
+  explicit BraveRewardsUI(content::WebUI* web_ui, const std::string& host);
```

Comment:
We don't need to use `explicit` keyword for ctor that has more than one parameter.

Reply:
oops left over from copying from somewhere.  Fixed.

Reply:
Oh.. I didn't think about initializer-list. Thanks for checking this!